api-tester/build.gradle

```diff
@@ -18,4 +18,5 @@ dependencies {
     implementation project(path: ':public')
     implementation project(path: ':purchases')
     implementation "androidx.annotation:annotation:$annotationVersion"
+    unityIAPCompileOnly "com.android.billingclient:billing:$billingClient3Version"
```

Comment:
why do we need this one?

Reply:
This one is needed because we are directly referencing classes like `SkuDetails` in the api tester, but does classes are not present when running the api testers in the unityIAP variant, since the billingclient3 dependency is compileOnly in the purchases module and not api.

If a developer integrates  the unityiap variant of the plugin, they would have to add a dependency to the billingclient themselves (which unity iap does automatically and that's why it gives duplicated classes issues). So this is basically doing that for the unityIAP variant.

I made it `compileOnly` but I think it could be `implementation`. I was just worried this module would somehow get downloaded by the consumers, which I am sure it doesn't, but it makes me feel better to have it as `compileOnly`

Reply:
ahhh got it, didn't realize we reference billingclient stuff. that all makes sense to me, and agree we should be as careful as possible with unity haha. thanks for explaining